### PR TITLE
feat: add css bem module loader

### PR DESCRIPTION
**Pull request by sanjaytwisk**
This PR adds a custom css module BEM loader to be used with `css-loader` in modules mode.

**Test**
- [ ] Run `npm run test` and confirm the tests are passing
- [ ] Any feedback regarding the readme is welcome

### CODE REVIEW

**Review by iksi — APPROVED** [review]
Looks good to me! Thanks for adding this 🙏🏻

File: packages/css-module-bem-loader/test/fixtures/invalid.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.invalid-block {
+  margin: 1rem;
+}
+
+.invalid-block__element {
+  margin: 1rem;
+}
+
+.invalid-block--modifire {
```
Comment (heldinz) [inline packages/css-module-bem-loader/test/fixtures/invalid.css:9]: Tiny typo: `modifier`

**Review by heldinz — APPROVED** [review]
Super nice! All I found is a typo 🙂

**Review by tomislav-s — APPROVED** [review]
🎉

**Pull request merged**